app/assets/javascripts/darkswarm/services/stripe_elements.js.coffee

```diff
@@ -0,0 +1,47 @@
+Darkswarm.factory 'StripeElements', ($rootScope, Loading, RailsFlashLoader) ->
+  new class StripeElements
+    # TODO: add locale here for translations of error messages etc. from Stripe
```

Comment:
Just spotted this - we should pass the locale in the calls to the Stripe API, which will ensure the credit card form and error messages are returned in the correct language. I couldn't figure out how to access it from the front end?

Reply:
Documented in #1864